docs/source/models/drinking_water_model_research.md

```diff
@@ -0,0 +1,977 @@
+
+### Reference
+- http://wateranalytics.org/EPANET/_pipes_page.html
+
+---
+
+## Question 7: Pump Status (open vs active vs closed) (Lines 257-261)
+
+**Q:** Can we model pump status with just `open` and `closed`?
+
+**A: Yes, with speed as a separate attribute.**
+
+### WNTR Pump States
+- **Active:** Pump is open AND has specific speed/setting applied
+- **Opened:** Pump is open at base speed (speed=1)
+- **Closed:** Pump is not operating
+
+### Simplified Modeling Approach
+Use two attributes:
+- `operational.status` (boolean): `True` = Open, `False` = Closed
+- `drinking_water.speed`: Speed multiplier (default 1.0)
+
+The "Active" state is effectively `status=True` + `speed != 1.0`
+
+### Reference
+- https://usepa.github.io/WNTR/apidoc/wntr.network.elements.Pump.html
+
+---
+
+## Question 8: Power Pump Speed (Lines 263-267)
+
+**Q:** Is speed ignored for power pumps? What does speed=1 mean?
+
+**A: Speed = 1.0 means nominal (design) operating speed.**
+
+### HeadPump Behavior
+- Speed scales the pump curve proportionally
+- Different speeds shift both flow and head characteristics
+- Formula: H = A - B*Q^C (coefficients derived from pump curve)
+
+### PowerPump Behavior (CONFIRMED FROM SOURCE CODE)
+- Primary characteristic is **fixed power value** stored as `_base_power`
+- **Speed is IGNORED for PowerPumps** - WNTR source code explicitly states:
+  ```python
+  link_res['setting'][name].append(1)  # power pumps have no speed
+  ```
+- PowerPumps operate at constant capacity without variable speed control
+- Speed attributes are inherited from parent class but not used in simulation
+
+### Reference
+- https://usepa.github.io/WNTR/apidoc/wntr.network.elements.HeadPump.html
+- https://usepa.github.io/WNTR/apidoc/wntr.network.elements.PowerPump.html
+- WNTR source: https://github.com/USEPA/WNTR/blob/main/wntr/sim/hydraulics.py
+
+---
+
+## Question 9: Valve Types Explained (Lines 303-305)
+
+**Q:** How do the different valve types work?
+
+### Valve Type Summary
+
+| Valve | Full Name | Purpose | Setting Unit |
+|-------|-----------|---------|--------------|
+| **PRV** | Pressure Reducing Valve | Limits downstream pressure to setting | Pressure (m) |
+| **PSV** | Pressure Sustaining Valve | Maintains upstream pressure at setting | Pressure (m) |
+| **PBV** | Pressure Breaker Valve | Maintains constant pressure DROP across valve | Pressure drop (m) |
+| **FCV** | Flow Control Valve | Limits maximum flow to setting | Flow (m³/s) |
+| **TCV** | Throttle Control Valve | Uses minor loss coefficient to restrict flow | Loss coefficient |
+| **GPV** | General Purpose Valve | User-defined head loss vs flow curve | Curve ID |
+
+### Placement Rules
+- PRVs cannot share same output node or be daisy-chained
+- PSVs cannot share same input node or be connected in series
+- PSV cannot connect to output of PRV
+- **PRV, PSV, FCV cannot connect directly to reservoir/tank** (use intermediate pipe)
```

Comment:
Primary concern is how do they interact with input/output data. Which ones have a "setting" parameter and what does the setting mean? Is the status only input or also output?